source/release-notes/2.1.0.rst

```diff
@@ -0,0 +1,87 @@
+.. _2.0.1:
+
+===========================================
+Percona Monitoring and Management 2.1.0
+===========================================
+
+:Date: November 11, 2019
+
+*PMM* (`Percona Monitoring and Management <https://www.percona.com/doc/2.x/percona-monitoring-and-management/index.html>`_) is a free and open-source platform for managing and monitoring |MySQL|, |MongoDB|, and |postgresql| performance. You can run *PMM* in your own environment for maximum security and reliability. It provides thorough time-based analysis for |MySQL|, |MongoDB|, and |postgresql| servers to ensure that your data works as efficiently as possible.
+
+For install instructions, see :ref:`deploy-pmm`.
+
+.. note:: PMM 2
+   is designed to be used as a new installation — please don’t try to upgrade
+   your existing PMM 1 environment.
+
+In this release, we have updated versions of the following core software PMM is
+based on:
+
+    Grafana 6.4.1
+    Go 1.12.10
```

Comment:
does anyone realy care about the number without  text ? Let's remove

Reply:
moved to the list of improvements (I remember there was some request to publish grafana version, so this may be valuable for someone)